crates/uv-resolver/src/exclude_newer.rs

```diff
@@ -425,47 +427,175 @@ impl std::fmt::Display for ExcludeNewerValue {
     }
 }
 
+/// Backwards-compatible alias for the exclude-newer timestamp type.
+pub type ExcludeNewerTimestamp = ExcludeNewerValue;
```

Comment:
Why do we have a backwards compatible alias? We don't need to maintain Rust API compatibility